torch/csrc/jit/passes/quantization.cpp

```diff
@@ -172,6 +172,12 @@ std::vector<size_t> getGeneralOpTensorInputIndexes(Node* n) {
                  /* call_funcs = */ {},
                  /* aten_funcs = */ single_input_aten_funcs)) {
     return {0};
+  } else if (n->kind() == prim::ListConstruct) {
```

Comment:
Can you add a test that exercises this code?

Reply:
this will be tested in https://github.com/pytorch/pytorch/pull/34346